rust/src/cliwrap.rs

```diff
@@ -74,7 +73,6 @@ pub fn entrypoint(args: &[&str]) -> Result<()> {
             "yum" | "dnf" => Ok(self::yumdnf::main(host_type, args)?),
             "dracut" => Ok(self::dracut::main(args)?),
             "grubby" => Ok(self::grubby::main(args)?),
-            "kernel-install" => Ok(self::kernel_install::main(args)?),
             _ => Err(anyhow!("Unknown wrapped binary: {}", name)),
```

Comment:
This error is triggered now, I am testing with this container file:

```
FROM quay.io/fedora/fedora-kinoite:40
ADD target/debug/rpm-ostree /usr/bin/rpm-ostree
RUN echo "layout=ostree" > /usr/lib/kernel/install.conf
RUN cat /usr/lib/kernel/install.conf
RUN rpm-ostree override replace https://koji.fedoraproject.org/koji/buildinfo?buildID=2561864
```

Trying to workaround that next.